7_react/react1/week1/homework/todo-list/src/components/todolist.jsx

```diff
@@ -0,0 +1,16 @@
+// TodoList.jsx
+import React from "react";
+import TodoItem from "./TodoItem";
+
+function TodoList() {
+  return (
+    <div>
+      <h2>Todo List</h2>
+      <TodoItem description="Get out of bed" deadline="Wed Sep 13 2017" />
```

Comment:
Great job with the components! Not mandatory but could you come up with another way to render multiple identical elements without copying and pasting them? 🤔 

Hint: The solution involves one of the JavaScript methods. 💪